Website/src/rendering/src/assets/css/components/Speakers/FeaturedSpeakers.css

```diff
@@ -14,6 +14,10 @@
 
       .item-details {
         @apply text-left p-0;
+
+        .item-title {
+          @apply font-bold my-1;
```

Comment:
font-bold is not needed as it is the default for all `item-title` in `grids.css`